test.js

```diff
@@ -204,7 +204,7 @@ test('assign styles', t => {
 });
 
 test('assign other props', t => {
-	const el = <a href="video.mp4" id="a" download referrerPolicy="no-referrer">Download</a>;
```

Comment:
Not sure of what you mean. `referrerpolicy` is a HTML prop and it's not camel case. In what occasion should prop differ from how they're defined?